server/chat-commands/moderation.ts

```diff
@@ -286,7 +286,7 @@ export const commands: ChatCommands = {
 		const buffer = [];
 		let innerBuffer = [];
 		const group = Users.globalAuth.get(targetId);
-		if (group) {
+		if (group !== ' ' || Users.isTrusted(targetId)) {
```

Comment:
```suggestion
		if (Users.globalAuth.has(targetId)) {
```

Reply:
n-nothing wrong with your code; I'm just golfing

Reply:
yeah, it looks like trusted users who aren't global auth aren't in `Users.globalAuth`